libparsec/crates/types/src/local_manifest.rs

```diff
@@ -87,7 +87,8 @@ impl Chunk {
             stop,
             raw_offset: start,
             // TODO: what to do with overflow
-            raw_size: NonZeroU64::try_from(stop.get() - start).unwrap_or_else(|_| unreachable!()),
+            raw_size: NonZeroU64::try_from(stop.get() - start)
+                .expect("Chunk `raw_size` integer should not be zero"),
             access: None,
         }
     }
```

Comment:
```suggestion
    }
 
```

Minor suggestion to improve the spacing between method